katello.te

```diff
@@ -49,8 +51,8 @@ require{
 # Allow FFI library callbacks - see RM#26520 for more info
 allow passenger_t self:process execmem;
 
-# katello does connect to AMQP service
-corenet_tcp_connect_amqp_port(passenger_t)
+# Candlepin Event Listener connects to Artemis
+allow passenger_t candlepin_activemq_port_t:tcp_socket name_connect;
```

Comment:
This appear to be correct, there is no macro exposed in `candlepin.if` therefore we need to use it this way.